src/libraries/pretest.proj

```diff
@@ -2,6 +2,8 @@
 
   <PropertyGroup>
     <TraversalGlobalProperties>BuildAllProjects=true</TraversalGlobalProperties>
+    <NETCoreAppFrameworkIdentifier>.NETCoreApp</NETCoreAppFrameworkIdentifier>
+    <SharedFrameworkName>Microsoft.NETCore.App</SharedFrameworkName>
```

Comment:
Share values with installer project tree? (Move to common ancestor?)

Reply:
Where would be a good place to place these properties in `<runtimeroot>/src/Directory.Build.props`?
Is it a significant impact to performance/organization to move it up? I'm not familiar enough to know now if those properties have a chance of changing or differing between the directories in `<runtimeroot>/src`. If they were to differ later would it be better to have each specify their own properties?

Reply:
It might not be worth it. My knee-jerk reaction is that duplicating code is what needs justification rather than reusing it, so just a quick suggestion.

Reply:
Let's open an issue for this. Consolidate all places in props and targets where we use Microsoft.NETCore.App to a single shared property.